pkg/k8s/cnp.go

```diff
@@ -448,6 +565,8 @@ func updateStatusesByCapabilities(client clientset.Interface, capabilities k8sve
 	}
 	// Updating succeeded - the updated map can be 'emptied' of updates that
 	// we need to propagate.
-	nodeStatuses = map[string]cilium_v2.CiliumNetworkPolicyNodeStatus{}
+	for k := range nodeStatuses {
+		delete(nodeStatuses, k)
+	}
```

Comment:
Why this change?

Reply:
`ineffassign` complains that I am doing an ineffectual assignment (which surprised me, since I am updating the map itself which is a reference). The compiler will optimize the "for all entries, delete" FWIW.